tools/RAiDER/losreader.py

```diff
@@ -197,6 +198,23 @@ def __init__(self, filename=None, los_convention='isce', time=None, look_dir = '
             self._look_dir = isce.core.LookSide.Left
         else:
             raise RuntimeError(f"Unknown look direction: {look_dir}")
+    
+
+    def getSensorDirection(self):
+        if self._orbit is None:
+            raise ValueError('The orbit has not been set')
+        z = self._orbit.position[:,2]
+        t = self._orbit.time
+        start = np.argmin(t)
+        end = np.argmax(t)
+        if z[start] > z[end]:
+            return 'desc'
+        else:
+            return 'asc'
```

Comment:
Does this not make ray-tracing break in case the LOS is provided as curbed. We plan to add that in future.

Reply:
> Does this not make ray-tracing break in case the LOS is provided as curbed. We plan to add that in future.

@dbekaert what is "curbed"? I'm not familiar with that term. This code relies on the fact that the orbit object provides x/y/z position for the sensor. As long as you have the orbit object you should have the x/y/z coordinates.

Reply:
That was a typo. He meant providing an LOS cube. Yes, this will break raytracing @dbekaert.
I have added this to the issue ticket regarding the LOS cube and we will have to address it when it's implemented, but for now I think this PR should be merged.